### PR TITLE
Legal Values cleanup

### PR DESCRIPTION
**Pull request by JeremyKennedy**
This PR removes the legacy "intermediary" object for accessing Legal Values, and simply accesses them directly from the JSON file. I've also cleaned up some of the naming conventions in the JSON file.

### CODE REVIEW

**Review by alex-solo — APPROVED** [review]
looks good

File: utils/api/benefits/entitlementFormula.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
   PartnerBenefitStatusHelper,
 } from '../helpers/fieldClasses'
 import roundToTwo from '../helpers/roundToTwo'
-import { legalValues } from '../scrapers/output'
+import legalValues from '../scrapers/output'
```
Comment (MarcoGoC) [inline utils/api/benefits/entitlementFormula.ts:8]: this line show as error on VS code.  
"... did you mean to use { } ... instead"

**Review by MarcoGoC — COMMENTED** [review]
weird VS code shows all these lines as error, maybe I am missing something  
  import legalValues from '../scrapers/output'

File: utils/api/benefits/entitlementFormula.ts (hunk 1 above)
Comment (JeremyKennedy) [reply utils/api/benefits/entitlementFormula.ts:8]: I think the VSCode errors is just a caching issue. If you load `/scrrapers/output/index.ts` it should update the VSCode error checking cache, I think!

**Pull request merged**